modules/common-fs2/src/test/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/EnrichSpec.scala

```diff
@@ -93,25 +96,29 @@ class EnrichSpec extends Specification with CatsIO with ScalaCheck {
       implicit val cpGen = PayloadGen.getPageViewArbitrary
       prop { (collectorPayload: CollectorPayload) =>
         val payload = collectorPayload.toRaw
-        SpecHelpers.createIgluClient(List(TestEnvironment.embeddedRegistry)).flatMap { igluClient =>
-          Enrich
-            .enrichWith(
-              TestEnvironment.enrichmentReg.pure[IO],
-              TestEnvironment.adapterRegistry,
-              igluClient,
-              None,
-              EnrichSpec.processor,
-              EnrichSpec.featureFlags,
-              IO.unit
-            )(
-              payload
-            )
-            .map(normalizeResult)
-            .map {
-              case List(Validated.Valid(e)) => e.event must beSome("page_view")
-              case other => ko(s"Expected one valid event, got $other")
-            }
-        }
+        SpecHelpers
+          .createIgluClient(List(TestEnvironment.embeddedRegistry))
+          .flatMap { igluClient =>
+            Enrich
+              .enrichWith(
+                TestEnvironment.enrichmentReg.pure[IO],
+                TestEnvironment.adapterRegistry,
+                igluClient,
+                None,
+                EnrichSpec.processor,
+                EnrichSpec.featureFlags,
+                IO.unit,
+                SpecHelpers.registryLookup
+              )(
+                payload
+              )
+              .map(normalizeResult)
+              .map {
+                case List(Validated.Valid(e)) => e.event must beSome("page_view")
+                case other => ko(s"Expected one valid event, got $other")
+              }
+          }
+          .unsafeRunSync()
```

Comment:
The old version didn't need this `unsafeRunsync()`.  How come you needed to add it?

Reply:
Good question, I'm not sure which dependency bump broke this, but there was an implicit missing. I tried using https://github.com/typelevel/scalacheck-effect but it didn't work out of the box and I thought that it was not worth it to spend time on it, although I don't like this `unsafeRunsync()`

Reply:
Ah I see -- it's something to do with the combination of using scalacheck with io.

That's fine.  I'm ok with leaving `unsafeRunSync()`.

Reply:
> it's something to do with the combination of using scalacheck with io

Exactly ! But we were already doing this before